config/__init__.py

```diff
@@ -93,6 +104,7 @@ def update_status_config(milestones_content):
     if not set(status_names).issubset(milestone_names):
         missing = set(status_names).symmetric_difference(set(milestone_names))
         print(MILESTONE_UNMATCHING.format(names=list(missing)))
+        milestones_content = MILESTONE_UNMATCHING
```

Comment:
This `milestones_content` value is scoped only to `update_status_config`, which means it has no meaning outside of this function. The important part which needs to be checked is the return value of the function. In this case: `None`.

Also note that we are only testing `not set(status_names).issubset(milestone_names)` in that `if`.